entity-types/ext-database/dashboard.json

```diff
@@ -0,0 +1,206 @@
+{
+  "name": "Database Overview",
+  "description": null,
+  "pages": [
+    {
+      "name": "Database Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Database State",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(stateDesc) FROM DatabaseStatusSample FACET name"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Unallocated Space",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(unallocatedSpace) AS 'Unallocated Space / MB' FROM DatabaseStatusSample"
```

Comment:
NIT: You can use units in dashboard NRQL too.
For example see here https://github.com/newrelic/entity-definitions/blob/2cdcaba85f8da60ee8785a2a9b4ca3516fe1b4e2/entity-types/infra-kubernetes_persistentvolumeclaim/dashboard.json#L256

Reply:
thanks, i added the units to the dashboard